python/pyspark/ml/feature.py

```diff
@@ -3754,6 +3755,23 @@ def _create_model(self, java_model: "JavaObject") -> Bucketizer:
         """
         Private method to convert the java_model to a Python model.
         """
+        if is_remote():
+            remote_model = JavaModel(java_model)
+            if self.isSet(self.inputCol):
+                return Bucketizer(
+                    splits=remote_model._call_java("getSplits"),
```

Comment:
`Bucketizer` is kind of special, it not only can be directly used as a normal transformer, but also can be fitted by a `QuantileDiscretizer`.

When fitted by `QuantileDiscretizer`, it fetches the coefficients and builds a client side model